ocw/lib/gce.py

```diff
@@ -10,6 +10,8 @@
 
 class GCE(Provider):
     __instances = {}
+    # XXX: Perhaps we should get this from pcw.ini
+    __skip_networks = frozenset({"default"})
```

Comment:
how this one works?

Reply:
> how this one works?

Testing for membership is faster on sets, and immutables sets are faster than vanilla sets.

Reply:
so this one is a sort of dummy initialization for performance in some following part of code. Right? Has TTL value from pcw.ini to be taken in account here or later? Which is following part of code involved? Is `is_outdated` ?

Reply:
This value is only for skipping networks in cleanup_xxx(), no TTL involved.

Reply:
DISCLAIMER : I am not against use of immutable sets here . I just want to raise more global topic not directly related to this PR .

> Testing for membership is faster on sets, and immutables sets are faster than vanilla sets.

This argument can not be used as a prove of something in context of PCW . We speaking about single threaded app which is doing one iteration in 20 minutes and set which we discussing will contain no more than 100 elements in very very bad situation . 
Such argument can be used in case of multi-threaded up which processing 10000000 elements in several threads at the same time. 
Let's always keep context of what we doing and why . Again maybe not directly related to this certain case but I have seen so many cases when people were spending few days to improve performance of test and make it run 1 hour 2 minutes instead of 1 hour 10 minutes .... 

TDLR Performance of code execution is certainly not the top prio for PCW and for most of projects which we maintaining ...

Reply:
> TDLR Performance of code execution is certainly not the top prio for PCW and for most of projects which we maintaining ...

I was about to add a `concurrent.futures.ThreadPoolExecutor()` to the delete calls for each region.

I'm joking.  XD